tests/pr_test.go

```diff
@@ -216,3 +219,60 @@ func TestRunUpgradeVpcPattern(t *testing.T) {
 		assert.NotNil(t, output, "Expected some output")
 	}
 }
+
+func TestRunOverride(t *testing.T) {
+	t.Parallel()
+
+	options := setupOptionsQuickStartPattern(t, "slz-ovr", quickStartPatternTerraformDir)
+	options.SkipTestTearDown = true
+	output, err := options.RunTestConsistency()
+
+	if assert.Nil(t, err, "This should not have errored") &&
+		assert.NotNil(t, output, "Expected some output") &&
+		assert.NotNil(t, options.LastTestTerraformOutputs, "Expected some Terraform outputs") {
+		// set override json string with previous value of config output
+		options.TerraformOptions.Vars["override_json_string"] = options.LastTestTerraformOutputs["config"]
+
+		options.SkipTestSetup = true
```

Comment:
Now that you are not using wrapper for 2nd plan, you don't need to set SkipTestSetup.